om/README.md

```diff
@@ -38,7 +38,7 @@ func main() {
 
     // lookup "my_prefix:01FNH4FCXV9JTB9WTVFAAKGSYB" through client side caching
     exp2, _ := repo.FetchCache(ctx, exp.Key, time.Second*5)
-    fmt.Println(exp2.Str) // output "mystr", which equals to exp.Str
+    fmt.Println(exp2.Str) // output "mystr", which is equal to exp.Str
```

Comment:
Either `equals` or `is equal to`, but never `equals to`.

I generally try to pick the one that feels closest to author intent, but it's pretty much a coin toss, this time I went with lengthening as opposed to shortening.